### PR TITLE
Debug statistics record score-set and keywords problem.

### PR DESCRIPTION
**Pull request by EstelleDa**
Model options are from Class RecordNames(experiment and score-set) and fields options are from Class RecordFields(publication-identifiers, keywords, doi-identifiers, raw-read-identifiers and created-by). 

However, score-set doesn't have keywords anymore. Hence, adding a validator to handle this case.

### CODE REVIEW

File: src/mavedb/routers/statistics.py [hunk 1]
```diff
@@ -260,6 +260,7 @@ def _record_from_field_and_model(
     model_created_by_field = getattr(queried_model, "created_by_id")
     model_published_data_field = getattr(queried_model, "published_date")
     if field is RecordFields.createdBy:
+        print("111")
```
Comment (bencap) [inline src/mavedb/routers/statistics.py:263]:
```suggestion
```

File: src/mavedb/routers/statistics.py [hunk 2]
```diff
@@ -269,6 +270,7 @@ def _record_from_field_and_model(
 
         return db.execute(query).all()
     else:
+        print("2222")
```
Comment (bencap) [inline src/mavedb/routers/statistics.py:273]:
```suggestion
```

**Review by bencap — APPROVED** [review]
Thanks for keeping an eye on the alerts channel Estelle, I appreciate it. If you can just remove the print statements before merging this looks good.

File: src/mavedb/routers/statistics.py (hunk 1 above)
Comment (EstelleDa) [reply src/mavedb/routers/statistics.py:263]: oops. Totally forgot them. Thanks Ben!

**Pull request merged**